docs/content/en/docs/yaml-crd-ref/evaluationdefinition.md

```diff
@@ -0,0 +1,110 @@
+---
+title: KeptnEvaluationDefinition
+description: Define all workloads and checks associated with an application
+weight: 20
+---
+---
+
+title: KeptnEvaluationDefinition
+description: Define all evaluations associated with an application
+weight: 30
+---
```

Comment:
```suggestion
---
title: KeptnEvaluationDefinition
description: Define all evaluations associated with an application
weight: 30
---
```

or something like that, i am not sure what is the right content, but it is duplicated currently

Reply:
Fixed -- thanks!